dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -36,15 +36,15 @@
     %h2= level.title
   %div= render(inline: level.markdown_instructions, type: :md)
 
-  - if level.allow_user_uploads
+  - if level.allow_user_uploads == 'true'
```

Comment:
`level.allow_user_uploads?` exists but [it return the same thing as `level.allow_user_uploads`](https://github.com/code-dot-org/code-dot-org/blob/8ee5d8fcf2a87dbc42004e23dee2d7af4c09567a/dashboard/app/models/concerns/serialized_properties.rb#L51) 😄 

I'll fix that method to use [`JSONValue.value`](https://github.com/code-dot-org/code-dot-org/blob/33286beaed328c7b3bb234ad3337c0c69cc3b604/dashboard/lib/json_value.rb#L15).